infer/src/base/Config.ml

```diff
@@ -2381,6 +2381,12 @@ and sarif =
     "Output issues in SARIF (Static Analysis Results Interchange Format) in infer-out/report.sarif"
 
 
+and no_sarif_codeflows =
+  CLOpt.mk_bool ~long:"no-sarif-codeflows" ~default:false
```

Comment:
Can this be called `sarif-codeflows` instead and default to `true`? This would avoid having a `--no-no-sarif-codeflows` option ;)

Reply:
Actually there's logic in CommandLineOption.ml to avoid generating "no-no-xxx" options but still it would be more direct (and more consistent with the other bool options) to define the positive option instead of the negative one.

Reply:
Sure. Let me update it to `drop-sarif-codeflows`

Reply:
Done

Reply:
Can this be `--sarif-codeflows` instead? One would pass `--no-sarif-codeflows` to disable it as in your original change, which I thought was good, I just want to avoid a negative definition (to avoid double negations in the code, eg `not Config.no_sarif_codeflows`).